benchmark/app.py

```diff
@@ -36,6 +22,8 @@ def index(yaml_path, num_docs, index_type='bytes', num_bytes_per_doc=10, num_chu
         pass
         
     f = Flow().load_config(filename=yaml_path)
```

Comment:
I thought that what we wanted is to have a `BenchmarkDriver` for every Pod? Am I wrong?

Reply:
I've changed the `BenchmarkDriver` as the `end_time` was getting updated in the `post_hook`, which makes the driver unable to read the updated envelope. Taking inspiration from `ReduceDriver`, I'm keeping track of all the old messages for all batches. Hence adding another pod at the last, which doesn't do anything else except using the `BenchmarkDriver`

Reply:
And this `BenchmarkDriver` will be responsible to count all the times per Pod?

Reply:
Yes.
e.g., For [this flow](https://github.com/jina-ai/stress-test/blob/feat-stresstest-benchmark-867/benchmark/yamls/index.yml), we'll get the following values for each batch - 
```
[
    'gateway_start_time', 'gateway_end_time', 
    'pod_0-head_start_time', 'pod_0-head_end_time', 
    'pod_0_start_time', 'pod_0_end_time',
    'pod_0-tail_start_time', 'pod_0-tail_end_time',
    'pod_1-head_start_time', 'pod_1-head_end_time', 
    'pod_1_start_time', 'pod_1_end_time'
    'pod_1-tail_start_time', 'pod_1-tail_end_time', 
    'benchmark_pod_start_time', 'benchmark_pod_end_time', 
]
```

Reply:
Perfect then! It looks really good